subgraph/schema.graphql

```diff
@@ -63,7 +63,7 @@ type Amm @entity {
   createdTransaction: Bytes!
   series: [SeriesAmm]! @derivedFrom(field: "amm")
   controller: SeriesController!
-  priceOracle: Bytes!
+  priceOracle: Bytes
```

Comment:
should we just get rid of this field, since it's not going to be set to anything

Reply:
we kept it for backwards compatibility I believe

Reply:
I don't think we need this, because we don't want front-end to be relying on a field that's never set

Reply:
okay I can remove and make sure the graph syncs

Reply:
yea, that's just for schema. It shouldn't create any issues with indexing/mappings. You can remove